app/src/main/java/com/amaze/filemanager/ui/activities/superclasses/PreferenceActivity.java

```diff
@@ -64,6 +65,7 @@ public void onCreate(final Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
   }
 
+  @NonNull
```

Comment:
Why?

Reply:
For kotlin, so that when you call the method it knows it returns `SharedPreferences` and not `SharedPreferences?`